server/game/PlayDisguisedCharacterAction.js

```diff
@@ -48,6 +49,9 @@ class DisguisedReduceableFateCost extends ReduceableFateCost {
     }
 
     getReducedCost(context) {
+        if(!context.costs.chooseDisguisedCharacter) {
+            return;
```

Comment:
I'm not sure what this is intended to achieve - you're just pushing a potential crash somewhere else...